### ISSUE TITLE
Error: [vite-plugin-ssr@0.4.40][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. 

### ISSUE THREAD

**Issue by dnyall** [brillout/vite-plugin-ssr#469]
### Description

when run dev server vite message me :
Server running at http://localhost:9000
Error: [vite-plugin-ssr@0.4.40][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Reach out at https://github.com/brillout/vite-plugin-ssr/issues/new or https://discord.com/invite/qTq92FQzKb and include this error stack (the error stack is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Do not hesitate to reach out as it makes vite-plugin-ssr more robust.
    at inferMediaType (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\inferMediaType.js:6:24)
    at ...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage\getPageAssets.js:38:94
    at Array.forEach (<anonymous>)
    at getPageAssets (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage\getPageAssets.js:37:15)
    at async Object._getPageAssets (...\bonak-tala\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:398:36)
    at async getHtmlSnippets (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:70:24)
    at async injectAtStreamBegin (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:46:24)
    at async injectAssets (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:30:18)
    at async renderHtml (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\renderHtml.js:36:26)
    at async executeRenderHook (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:557:24)
    at async renderPageContext (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:88:30)
    at async renderPage (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:155:16)
    at async \server\index.js:32:23

please help.

### Error Stack

```shell
Error: [vite-plugin-ssr@0.4.40][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Reach out at https://github.com/brillout/vite-plugin-ssr/issues/new or https://discord.com/invite/qTq92FQzKb and include this error stack (the error stack is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Do not hesitate to reach out as it makes vite-plugin-ssr more robust.
    at inferMediaType (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\inferMediaType.js:6:24)
    at ...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage\getPageAssets.js:38:94
    at Array.forEach (<anonymous>)
    at getPageAssets (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage\getPageAssets.js:37:15)
    at async Object._getPageAssets (...\bonak-tala\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:398:36)
    at async getHtmlSnippets (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:70:24)
    at async injectAtStreamBegin (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:46:24)
    at async injectAssets (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\injectAssets.js:30:18)
    at async renderHtml (...\node_modules\vite-plugin-ssr\dist\cjs\node\html\renderHtml.js:36:26)
    at async executeRenderHook (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:557:24)
    at async renderPageContext (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:88:30)
    at async renderPage (...\node_modules\vite-plugin-ssr\dist\cjs\node\renderPage.js:155:16)
    at async \server\index.js:32:23
```

**Comment by brillout** [comment]
Add `console.log(href)` before `node_modules\vite-plugin-ssr\dist\cjs\node\html\inferMediaType.js:6` and let me know what you get.

**Comment by brillout** [comment]
Reproduction: https://discord.com/channels/815937377888632913/815937377888632916/1031932446649897071.

**Comment by brillout** [comment]
Fix released in `0.4.42`.
